frontend/.storybook/preview.js

```diff
@@ -0,0 +1,264 @@
+import React from "react"
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { store } from '../src/redux/store/store';
+import SDKInitialize from '../src/SDKInitialize';
+import { BrowserStorageContextProvider } from '../src/components/browserStorage/BrowserStorageContext';
+import { initialize, mswDecorator } from 'msw-storybook-addon';
+
+import '@patternfly/patternfly/patternfly.min.css';
+import '@patternfly/patternfly/patternfly-addons.css';
+import { AppContext } from "../src/app/AppContext";
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+}
+
+// Initialize MSW
+initialize();
+
+export const decorators = [
+  (Story) => (
+    <AppContext.Provider
+      value={{
+        dashboardConfig: {
+          "apiVersion": "opendatahub.io/v1alpha",
+          "kind": "OdhDashboardConfig",
+          "metadata": {
+            "name": "odh-dashboard-config",
+            "labels": {
+              "opendatahub.io/dashboard": "true"
+            },
+            "annotations": {
+              "kubectl.kubernetes.io/last-applied-configuration": "{\"apiVersion\":\"opendatahub.io/v1alpha\",\"kind\":\"OdhDashboardConfig\",\"metadata\":{\"annotations\":{},\"creationTimestamp\":null,\"name\":\"odh-dashboard-config\",\"namespace\":\"redhat-ods-applications\"},\"spec\":{\"dashboardConfig\":{\"disableBYONImageStream\":false,\"disableClusterManager\":false,\"disableISVBadges\":false,\"disableInfo\":false,\"disableModelServing\":false,\"disableProjects\":false,\"disableSupport\":false,\"disableTracking\":false,\"enablement\":true},\"groupsConfig\":{\"adminGroups\":\"rhods-admins\",\"allowedGroups\":\"system:authenticated\"},\"modelServerSizes\":[{\"name\":\"Small\",\"resources\":{\"limits\":{\"cpu\":\"2\",\"memory\":\"8Gi\"},\"requests\":{\"cpu\":\"1\",\"memory\":\"4Gi\"}}},{\"name\":\"Medium\",\"resources\":{\"limits\":{\"cpu\":\"8\",\"memory\":\"10Gi\"},\"requests\":{\"cpu\":\"4\",\"memory\":\"8Gi\"}}},{\"name\":\"Large\",\"resources\":{\"limits\":{\"cpu\":\"10\",\"memory\":\"20Gi\"},\"requests\":{\"cpu\":\"6\",\"memory\":\"16Gi\"}}}],\"notebookController\":{\"enabled\":true,\"notebookNamespace\":\"rhods-notebooks\",\"pvcSize\":\"20Gi\"},\"notebookSizes\":[{\"name\":\"Small\",\"resources\":{\"limits\":{\"cpu\":\"2\",\"memory\":\"8Gi\"},\"requests\":{\"cpu\":\"1\",\"memory\":\"8Gi\"}}},{\"name\":\"Medium\",\"resources\":{\"limits\":{\"cpu\":\"6\",\"memory\":\"24Gi\"},\"requests\":{\"cpu\":\"3\",\"memory\":\"24Gi\"}}},{\"name\":\"Large\",\"resources\":{\"limits\":{\"cpu\":\"14\",\"memory\":\"56Gi\"},\"requests\":{\"cpu\":\"7\",\"memory\":\"56Gi\"}}},{\"name\":\"X Large\",\"resources\":{\"limits\":{\"cpu\":\"30\",\"memory\":\"120Gi\"},\"requests\":{\"cpu\":\"15\",\"memory\":\"120Gi\"}}}]}}\n"
```

Comment:
Can we not trim this to readable values instead of jamming in the k8s nonsense?

Ie, delete `kubectl.kubernetes.io/last-applied-configuration` and `managedFields`

Reply:
Yes we probably can. This is jsut the raw api response.